src/commands/skullboard.py

```diff
@@ -150,3 +167,97 @@ async def edit_or_send_skullboard_message(
         else:
             skullboard_message = await channel.fetch_message(skullboard_message_id)
             await skullboard_message.edit(content=message_content, embed=embed)
+
+
+class SkullGroup(app_commands.Group):
+    def __init__(self):
+        super().__init__(name="skull", description="Skullboard queries")
+        self.db = SkullboardDB()
+        self.skullboard_channel_id = int(str(os.environ.get("SKULLBOARD_CHANNEL_ID")))
+
+    @app_commands.command(name="about", description="Learn about the Skullboard")
+    async def about(self, interaction: Interaction):
+        skullboard_info = (
+            "## 💀 WELCOME TO THE SKULLZONE 💀\n\n"
```

Comment:
I think something like 
"💀 WELCOME TO THE BONEZONE 💀
The Skullboard is the way we track the BONIEST of BOZOS in the CS Club! 💀"
would be better.